services/fee-estimator/shared/utils/dynamicFees.js

```diff
@@ -13,42 +13,110 @@
  * Removal or modification of this copyright notice is prohibited.
  *
  */
-const { Logger } = require('lisk-service-framework');
+const BluebirdPromise = require('bluebird');
+
+const {
+	CacheRedis,
+	Logger,
+} = require('lisk-service-framework');
+
+const {
+	getEstimateFeeByteForBlock,
+} = require('./dynamicFeesLIP');
+const { requestConnector } = require('./request');
 
 const config = require('../../config');
 
+const executionStatus = {
+	// false: not running, true: running
+	[config.cacheKeys.cacheKeyFeeEstFull]: false,
+	[config.cacheKeys.cacheKeyFeeEstQuick]: false,
+};
+
+const cacheRedisFees = CacheRedis('fees', config.endpoints.cache);
 const logger = Logger();
 
-const calcAvgFeeByteModes = {
-	MEDIUM: 'med',
-	HIGH: 'high',
-};
+const getEstimateFeeByteForBatch = async (fromHeight, toHeight, cacheKey) => {
```

Comment:
Wasn't this supposed to be `getEstimateFeePerByteForBatch`?
```suggestion
const getEstimateFeePerByteForBatch = async (fromHeight, toHeight, cacheKey) => {
```

Reply:
Applied.